Zend/tests/stack_limit_001.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+Stack limit 001
+--FILE--
+<?php
+
+class Test1 {
+    public function __destruct() {
+        new Test1;
+    }
+}
+
+class Test2 {
+    public function __clone() {
+        clone $this;
+    }
+}
+
+function replace() {
+    return preg_replace_callback('#.#', function () {
+        return replace();
+    }, 'x');
+}
+
+try {
+    new Test1;
+} catch (Error $e) {
+    echo $e->getMessage(), "\n";
+}
+
+try {
+    clone new Test2;
+} catch (Error $e) {
+    echo $e->getMessage(), "\n";
+}
+
+try {
+    replace();
+} catch (Error $e) {
+    echo $e->getMessage(), "\n";
+}
+
+?>
+--EXPECTF--
+Maximum call stack size reached. Infinite recursion?
```

Comment:
Would it be possible to print here more details, especially the stack size and a reentry count?